cmd/fs-v1.go

```diff
@@ -828,7 +827,6 @@ func (fs *FSObjects) putObject(ctx context.Context, bucket string, object string
 		fsMetaPath := pathJoin(bucketMetaDir, bucket, object, fs.metaJSONFile)
 		wlk, err = fs.rwPool.Create(fsMetaPath)
 		if err != nil {
-			logger.LogIf(ctx, err)
```

Comment:
With my basic testing, I didn't see this is logged at all. Is this something we need to really ignore ?

Reply:
Same comment as above. Can keep it for now, and make a decision on this later.